builder/src/build/patches/investigated/update-webpack-chunks-file/get-chunk-installation-identifiers.ts

```diff
@@ -0,0 +1,77 @@
+import * as ts from "ts-morph";
+
+export async function getChunkInstallationIdentifiers(sourceFile: ts.SourceFile): Promise<{
+  installedChunks: string;
+  installChunk: string;
+}> {
+  const installChunkDeclaration = getInstallChunkDeclaration(sourceFile);
```

Comment:
Is it possible that there might be some false positives for "install chunk" declarations?
If so, could we cover ourselves a bit tighter by returning all of them here and then filtering them by the ones that have a matching "installed chunks" declaration directly in front?
And then if there is still more than one either erroring or at least warning?

Reply:
yes it's definitely possible to have false positives, but I think that it's very unlikely since we do check for some pretty specific things like the fact that the function has a single parameter and that it is used to access three of its properties in a specific order (`modules`, `ids` and `runtime`).

I feel like it's going to be pretty difficult to match those in non `installChunk` functions.

I can totally add the check you mentioned if you want but I feel that the extra complexity it'd introduce (even if not huge) would outweight the benefit it'd provide (personally I'd go with this and update the various checks if and when there's a need)